test/e2e/adminUI/pages/app.js

```diff
@@ -67,36 +67,40 @@ module.exports = {
 		targetrelationshipListSubmenu: '.secondary-navbar [data-list-path="target-relationships"]',
 	},
 	commands: [{
+		gotoHomeScreen: function() {
+			return this
+				.navigate();		// navigate to the configure Url
+		},
 		openMiscList: function(list) {
 			var list = list.toLowerCase() + 'List';
 			var listSubmenu = '@' + list + 'Submenu';
 			return this.click('@miscListsMenu')
-				.waitForElementVisible('@listScreen')
+				.waitForListScreen()
 				.click(listSubmenu)
-				.waitForElementVisible('@listScreen');
+				.waitForListScreen();
 		},
 		openFieldList: function(field) {
 				var list = field.toLowerCase() + 'List';
 				var listSubmenu = '@' + list + 'Submenu';
 				return this.click('@fieldListsMenu')
-					.waitForElementVisible('@listScreen')
+					.waitForListScreen()
 					.click(listSubmenu)
-					.waitForElementVisible('@listScreen');
+					.waitForListScreen();
 		},
 		signout: function() {
 			this.api.pause(500);
 			return this
 				.waitForElementVisible('@logoutIcon')
 				.click('@logoutIconLink')
-				.waitForElementVisible('@signinScreen');
+				.waitForSigninScreen();
 		},
 		waitForSigninScreen: function() {
 			return this
-				.waitForElementVisible('@signinScreen', 20000);
+				.waitForElementVisible('@signinScreen', 60000);
```

Comment:
I didn't mean to change `signin` to `list. What I meant was, this would be better as `.waitForSigninScreen(6000)`, and then update `waitForSigninScreen` to be
```
waitForSigninScreen: function(time) {
              return this
                 .waitForElementVisible('@signinScreen', time);
```